app/assets/javascripts/components/student/student_preview.es6.jsx

```diff
@@ -27,7 +27,9 @@ class StudentPreview extends Component {
     var student = this.props.student;
     return (
       <div style={this.styles.container}>
-        {student.first_name + ' ' + student.last_name}
+        <Clickable
+          content={student.first_name + ' ' + student.last_name}
```

Comment:
`student.first_name + ' ' + student.last_name`
can be done succinctly with
`\`student.first_name + ' ' + student.last_name`\`